gitlab-pipeline/stage/post.yml

```diff
@@ -43,3 +43,40 @@ coveralls:finish-build:
   script:
     - 'curl -k ${COVERALLS_WEBHOOK_URL}?repo_token=${MENDER_COVERALLS_TOKEN} -d "payload[build_num]=$(git rev-parse HEAD)&payload[status]=done"'
 
+.trigger_template:
+  stage: .post
+  rules:
+    # - if: '$INTEGRATION_REV =~ /^[0-9]+\.[0-9]+\.[0-9]+(-build\d+)?$/'
+    - when: always
+    # the following is to prevent an endless loop of qa pipelines caused by downstream pipelines
+    - if: '$CI_PIPELINE_SOURCE == "pipeline"'
+      when: never
+
+trigger:mender-convert:
+  extends: .trigger_template
+  variables:
+    MENDER_ARTIFACT_VERSION: $MENDER_ARTIFACT_REV
+    MENDER_CLIENT_VERSION: $MENDER_REV
+    MENDER_ADDON_CONNECT_VERSION: $MENDER_CONNECT_REV
+    MENDER_ADDON_CONFIGURE_VERSION: $DEVICECONFIG_REV
+  trigger:
+    project: Northern.tech/Mender/mender-convert
+    branch: pr_356
+    strategy: depend
+
+trigger:mender-binary-delta:
+  extends: .trigger_template
+  variables:
+    MENDER_VERSION: $MENDER_REV
+    MENDER_ARTIFACT_VERSION: $MENDER_ARTIFACT_REV
+  trigger:
+    project: Northern.tech/Mender/mender-binary-delta
+    branch: pr_92
+    strategy: depend
+
+trigger:mender-connect:
+  extends: .trigger_template
+  trigger:
+    project: Northern.tech/Mender/mender-connect
+    branch: pr_32
+    strategy: depend
```

Comment:
As commented on the call, this should not be necessary: `mender-connect` does not need to be triggered from `mender-qa` (if any, we would trigger `mender-dist-packages` directly but currently this is happening on a different by just tagging).

Reply:
👌 removed

Reply:
> As commented on the call, this should not be necessary: `mender-connect` does not need to be triggered from `mender-qa` (if any, we would trigger `mender-dist-packages` directly but currently this is happening on a different by just tagging).

But we don't see the `mender-dist-packages` pipeline here unless we trigger it from here, and that has a lot of value, don't you think?